### PR TITLE
Update ADV_STABLE_BASELINES_3.md

### PR DESCRIPTION
**Pull request by Ivan-267**
Updated the docs for the sb3 example. Removed the code snippet since the example file is already linked, it may be easier than keeping the code snippet updated with each change to the example.

### CODE REVIEW

File: docs/ADV_STABLE_BASELINES_3.md [hunk 1]
```diff
@@ -46,42 +46,66 @@ While the default options for sb3 work reasonably well. You may be interested in
...
-parser.add_argument(
-    "--env_path",
-    # default="envs/example_envs/builds/JumperHard/jumper_hard.x86_64",
-    default=None,
-    type=str,
-    help="The Godot binary to use, do not include for in editor training",
-)
+### Train an exported environment using 8 times speedup:
+```bash
+python .\stable_baselines3_example.py --env_path=path_to_executable --speedup=8
```
Comment (edbeeching) [inline docs/ADV_STABLE_BASELINES_3.md:72]: Do you need to include the "./"  in front of every command?

**Review by edbeeching — APPROVED** [review]
LGTM apart for the `.\` , is this a windows thing?

**Comment by Ivan-267** [comment]
> LGTM apart for the `.\` , is this a windows thing?

I'll check if it works without it, it might be needed in Windows Powershell (used with Anaconda) as it autocompletes for me that way (using python then pressing tab). If it's not needed, I'll remove it, but if it is needed I'll have to think about a solution to address both cases. Does it work that way in Linux?

**Comment by edbeeching** [comment]
Ye in linux it would just be:
```bash
python stable_baselines3_example.py --env_path=path_to_executable --speedup=8
```

**Comment by Ivan-267** [comment]
> Ye in linux it would just be:
> 
> ```shell
> python stable_baselines3_example.py --env_path=path_to_executable --speedup=8
> ```

Thanks for noticing this, I made a quick check and it seems to be Windows Powershell specific, so it wouldn't work for other OS. However, it works without the extra `./` in Powershell as well. I updated the docs to resolve it.

**Pull request merged**